src/foam/nanos/auth/User.js

```diff
@@ -224,10 +224,9 @@ foam.CLASS({
       documentation: 'The first name of the User.',
       section: 'userInformation',
       order: 70,
-gridColumns: { columns: 4, smColumns: 12, xsColumns: 12 },
+      gridColumns: { columns: 4, smColumns: 12, xsColumns: 12 },
       includeInDigest: true,
-      containsPII: true,
-      trim:true
```

Comment:
I don't understand why we're doing this. Why should we allow a space? We allow space in names, like “Kevin Glen Roy” but we trim off leading and trailing spaces so we don't allow " Kevin".

Reply:
Trim should not be the issue, or trim is broken - removing spaces from within words.

Reply:
Because trim will cut spaces around
So you can have “Kevin Glen Roy” but you need to type: "KevinGlenRoy" and then insert space